docs/update-levelbuilder.md

```diff
@@ -3,14 +3,14 @@ This .md file should only contain information which is specific to Code.org engi
 
 # How to update level builder
 
-## To update levelbuilder to match staging:
+## To update levelbuilder to match test:
 
-1. On GitHub, open a pull request from `staging` into `levelbuilder`, link: [levelbuilder...staging](https://github.com/code-dot-org/code-dot-org/compare/levelbuilder...staging?expand=1)
+1. On GitHub, open a pull request from `test` into `levelbuilder`, link: [levelbuilder...test](https://github.com/code-dot-org/code-dot-org/compare/levelbuilder...test?expand=1)
   1. Or in your local repository:
```

Comment:
I'd say it's so strongly preferred, that there's no need to document the alternative here.  It's reasonable to expect devs to look up what the "merge" button on GitHub does if they need to do it manually.  Updated.